frontend/components/login.js

```diff
@@ -100,43 +101,51 @@ function Login(props) {
                   setModalVisible(false);
                 }}
               >
-              <View style={[styles.centeredView, state.modalVisible ? {backgroundColor: 'rgba(0,0,0,0.5)'} : '']}>
+              <View style={[styles.centeredView, modalVisible ? {backgroundColor: 'rgba(0,0,0,0.5)'} : '']}>
               <View style={styles.modalView}>
-              <Text style={styles.logoText}>Sign Up</Text>
-             
-              <TextInput 
-                placeholder="Email" 
-                placeholderColor="#c4c3cb" 
-                style={styles.signupFormTextInput} 
+              <Icon
+            name="close"
+            size={30}
+            color="black"
+            onPress={() => {
+              setModalVisible(false);
+            }}
+            style={styles.modalExit}
+          />
```

Comment:
fix indent of the properties i.e:
   
              <TextInput
                placeholder="Email"
                placeholderColor="#c4c3cb"
                style={styles.signupFormTextInput}
              />

Reply:
ignore this comment! testing webhook